libs/cms/src/lib/models/articleReference.ts

```diff
@@ -31,6 +32,9 @@ export class ArticleReference {
   @Field()
   intro?: string
 
+  @CacheField(() => [SliceUnion], { nullable: true })
+  body?: Array<typeof SliceUnion> = []
```

Comment:
Why are we adding the body field to the article reference model? I thought the article reference was only there to prevent potential circularity between subArticle <-> article.

Reply:
The Featured Article model contains a collection of Article References. This was meant to be able to check if these articles contain a Process Entry type embeded in the rich text in addition to the main Process Entry filed to determine if they are Applications or not. I am unaware of what the original intent was but this slice's articles are currently populated with these references.

Reply:
I'm wondering another approach might be better, like if all we are using the body field for is simply to check if there is a process entry connected to this article, perhaps adding a boolean field instead that indicates if there is a process entry or not might be better?

Reply:
Agreed. In fact, I think this would be best in general for all Articles, not just here as I do not really like the current way the code checks for the presence of process entries. I'll remove this and address this in a separate PR as it doesn't really affect the current over view.